app/assets/stylesheets/pages/_result.scss

```diff
@@ -30,4 +30,22 @@
 
 #impostor-result{
     font-size: 32px;
-}
\ No newline at end of file
+}
+
+#most-yes-result {
+    background-color: $pink;
+    border: 2px solid #151f52;
+    border-radius: 8px;
+    border-style: groove;
+    margin: 8px;
+    text-align: center;
+}
+
+#most-no-result {
+    background-color: $mizu;
+    border: 2px solid #151f52;
+    border-radius: 8px;
+    margin: 8px;
+    text-align: center;
```

Comment:
@TofuSupreme sorry Shante I also just noticed the the colors the the other way around.
No - pink
Yes - mizu